contracts/contracts/automation/PlumeBridgeHelperModule.sol

```diff
@@ -0,0 +1,291 @@
+{
+    IVault public constant vault =
+        IVault(0xc8c8F8bEA5631A8AF26440AF32a55002138cB76a);
+    IWETH9 public constant weth =
+        IWETH9(0xca59cA09E5602fAe8B629DeE83FfA819741f14be);
+    IERC20 public constant oethp =
+        IERC20(0xFCbe50DbE43bF7E5C88C6F6Fb9ef432D4165406E);
+    IERC4626 public constant bridgedWOETH =
+        IERC4626(0xD8724322f44E5c58D7A815F542036fb17DbbF839);
+
+    uint32 public constant LZ_ETHEREUM_ENDPOINT_ID = 30101;
+    IOFT public constant LZ_WOETH_OMNICHAIN_ADAPTER =
+        IOFT(0x592CB6A596E7919930bF49a27AdAeCA7C055e4DB);
+    IOFT public constant LZ_ETH_OMNICHAIN_ADAPTER =
+        IOFT(0x4683CE822272CD66CEa73F5F1f9f5cBcaEF4F066);
+
+    BridgedWOETHStrategy public constant bridgedWOETHStrategy =
+        BridgedWOETHStrategy(0x1E3EdD5e019207D6355Ea77F724b1F1BF639B569);
+
+    constructor(address _safeContract)
+        AbstractLZBridgeHelperModule(_safeContract)
+    {}
+
+    /**
+     * @dev Bridges wOETH to Ethereum.
+     * @param woethAmount Amount of wOETH to bridge.
+     * @param slippageBps Slippage in 10^4 basis points.
+     */
+    function bridgeWOETHToEthereum(uint256 woethAmount, uint256 slippageBps)
+        external
+        payable
+        onlyOperator
+    {
+        _bridgeTokenWithLz(
+            LZ_ETHEREUM_ENDPOINT_ID,
+            IERC20(address(bridgedWOETH)),
+            LZ_WOETH_OMNICHAIN_ADAPTER,
+            woethAmount,
+            slippageBps,
+            false
+        );
+    }
+
+    /**
+     * @dev Bridges wETH to Ethereum.
+     * @param wethAmount Amount of wETH to bridge.
+     * @param slippageBps Slippage in 10^4 basis points.
+     */
+    function bridgeWETHToEthereum(uint256 wethAmount, uint256 slippageBps)
+        external
+        payable
+        onlyOperator
+    {
+        _bridgeTokenWithLz(
+            LZ_ETHEREUM_ENDPOINT_ID,
+            IERC20(address(weth)),
+            LZ_ETH_OMNICHAIN_ADAPTER,
+            wethAmount,
+            slippageBps,
+            false
+        );
+    }
+
+    /**
+     * @dev Deposits wOETH into the bridgedWOETH strategy.
+     * @param woethAmount Amount of wOETH to deposit.
+     * @param redeemWithVault Whether to redeem with Vault.
+     * @return Amount of OETHp received.
+     */
+    function depositWOETH(uint256 woethAmount, bool redeemWithVault)
+        external
+        onlyOperator
+        returns (uint256)
+    {
+        return _depositWOETH(woethAmount, redeemWithVault);
+    }
+
+    /**
+     * @dev Deposits wOETH into the bridgedWOETH strategy and bridges it to Ethereum.
+     * @param woethAmount Amount of wOETH to deposit.
+     * @param slippageBps Slippage in 10^4 basis points.
+     * @return Amount of WETH received.
+     */
+    function depositWOETHAndBridgeWETH(uint256 woethAmount, uint256 slippageBps)
+        external
+        onlyOperator
+        returns (uint256)
+    {
+        uint256 wethAmount = _depositWOETH(woethAmount, true);
+        _bridgeTokenWithLz(
```

Comment:
you could change the `bridgeWETHToEthereum` to a public function and replace this bit by just calling: 
`bridgeWETHToEthereum(wethAmount, slippageBps)`

Reply:
That's a really good suggestion, have cleaned up all instances